.github/workflows/main.yml

```diff
@@ -3,10 +3,10 @@ name: Claude Code Review
 on:
   pull_request:
     types: [opened, edited]
-  pull_request_review_comment:
-    types: [created, edited]
-  issue_comment:
```

Comment:
The code patch removes event triggers for pull request review comments and issue comments. This could potentially limit the functionality of the GitHub Action, as it will no longer respond to these events.
```yaml
# Consider keeping these event triggers if their functionality is still needed:
pull_request_review_comment:
  types: [created, edited]
issue_comment:
  types: [created, edited]
```